src/css/flags/touch.less

```diff
@@ -21,5 +21,8 @@
     .jw-display-icon-container {
         pointer-events: none;
     }
-}
 
+    &.jwplayer&.jw-state-paused .jw-display-icon-container {
```

Comment:
Understood!

I was hinting that it's more complex than it probably ought to be.

If for some reason it needs to be that "specific" then the alternative is making the rule you are overwriting less "specific" instead.